test/visualTest.ts

```diff
@@ -331,6 +331,24 @@ module powerbi.extensibility.visual.test {
                         });
                 });
             });
+
+            describe("In visual with small size", () => {
```

Comment:
Please rename this test to provide more readability. For instance: `Data labels at small viewport`.